src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -141,8 +153,14 @@ export const roomCreateSchema = Yup.object().shape<RoomSchemaShape>({
       is: false,
       then: Yup.string()
         .required("Venue name is required")
-        .min(3, ({ min }) => `Name must be at least ${min} characters`)
-        .max(25, ({ max }) => `Name must be less than ${max} characters`),
+        .min(
+          VENUE_NAME_MIN_CHAR_COUNT,
+          ({ min }) => `Name must be at least ${min} characters`
+        )
+        .max(
+          VENUE_NAME_MAX_CHAR_COUNT,
+          ({ max }) => `Name must be less than ${max} characters`
+        ),
     })
```

Comment:
Why are we duplicating the logic here rather than reusing a 'schema chunk' such as `roomTitleSchema`?

Reply:
See changes in https://github.com/sparkletown/sparkle/pull/1341